src/scripts/Player.js

```diff
@@ -1,7 +1,25 @@
+import $ from 'jquery';
+
 class Player {
   constructor() {
-    
+    this.totalPoints = 0;
+    this.name;
+    this.playerNum;
+    this.incorrectGuesses = [];
+    this.multiplier;
+  }
+  makeGuess() {
+
+  }
+  addPoints() {
+
+  }
+  setMultiplier() {
+
+  }
+  clearIncorrectGuess() {
```

Comment:
When we discussed our methods for the Player class, clearIncorrectGuess wasn't one we discussed, are we clearing the player's incorrect guesses for each round?